app/tasks/raster_tile_set_assets/utils.py

```diff
@@ -225,3 +225,33 @@ async def create_resample_job(
         parents=[parent.job_name for parent in parents] if parents else None,
         **kwargs,
     )
+
+async def create_unify_projection_job(
+    dataset: str,
+    old_source_uris: List[str],
+    target_prefix: str,
+    target_crs: str,
+    job_name: str,
+    callback: Callback
+):
+    """
+    """
```

Comment:
Can you add a function comment here?  Mainly would be that target_prefix is where the folders of warped files are placed, under SRC_0, SRC_1, SRC_2, etc.